server/tests/user.signin.test.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable import/no-extraneous-dependencies */
+// import regeneratorRuntime from 'regenerator-runtime';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../app';
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+describe('~~~ Testing Route (User Sign-In): POST /api/v1/users/signin ~~~', () => {
+  // User exists, and is verified
```

Comment:
@fordarnold takes advantage of the before() hooks to define variables, insert/load data into the database (without using calling the API) which these tests suits shall require and truncate the database.

Reply:
Seen.

- https://mochajs.org/#hooks